src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SceneFlameGraph.tsx

```diff
@@ -94,18 +112,20 @@ export class SceneFlameGraph extends SceneObjectBase<SceneFlameGraphState> {
     const profileData = $dataState?.data?.series?.[0];
     const hasProfileData = Number(profileData?.length) > 1;
 
-    const serviceName = getSceneVariableValue(this, 'serviceName');
-    const profileMetricId = getSceneVariableValue(this, 'profileMetricId');
-    const profileMetricType = getProfileMetric(profileMetricId as ProfileMetricId).type;
+    const query = useBuildPyroscopeQuery(this, 'filters');
 
     return {
       data: {
-        title: `${serviceName} flame graph (${profileMetricType})`,
+        title: this.buildTitle(),
         isLoading: isFetchingProfileData,
         isFetchingProfileData,
         hasProfileData,
         profileData,
         settings,
+        ai: {
+          panel: aiPanel,
+          fetchParams: [{ query, timeRange: lastTimeRange }],
```

Comment:
It's now easier to understand where the last time range is used and what for (vs calling `  timelineAndProfileApiClient.setLastTimeRange(lastTimeRange)` for the GitHub integration feature).